tests/test_models.py

```diff
@@ -33,3 +78,30 @@ def test_grad(self, basic_stim):
         L = po.simul.Laplacian_Pyramid().to(DEVICE)
         y = L.analysis(basic_stim)
         assert y[0].requires_grad
+
+
+class TestFilters:
```

Comment:
yeah, I like this way of testing the kernels. 

but this doesn't test anything about the frontend models' learnability. Can we do something to test that it doesn't end up with a broken graph or the like?

Reply:
oh wait, that's what the `test_gradient_flow` does, right? nevermind, ignore this